src/templates/base_register.html

```diff
@@ -79,15 +78,14 @@
     })
 });
 </script>
-
-{% endblock %}
-{% block extrajs %}
-{% endblock %}
+{% endblock js %}
```

Comment:
Are you sure that you want to end with block name?
We use this approach when we have nesting of blocks but we don't have any nesting here. If you think that we may need to add nesting in future, I admire your foreseeing. You should go even further and add block name in all endblocks. 

Ps: sed and grep are always there for yoy.